indy-wrapper/src/main/java/nl/quintor/studybits/indy/wrapper/dto/Proof.java

```diff
@@ -0,0 +1,21 @@
+package nl.quintor.studybits.indy.wrapper.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@Data
+@NoArgsConstructor
+public class Proof implements Serializable, AuthCryptable {
```

Comment:
Maybe rename to `ProofRecord` since you have both, a Proof and a ProofRequest in here.

Reply:
Note that the requestedProof is not actually the "ProofRequest" that went over the line earlier. It is some datastructure that contains similar data, but not in the same format. As such, it is really part of the proof, rather than an entity by itself. 

With this explanation, do you feel it still makes sense to rename?